### PR TITLE
Added bio.tools ref to mcl

### PR DESCRIPTION
**Pull request by RZ9082**
This PR links mcl to its bio.tools [entry](https://bio.tools/mcl).

### CODE REVIEW

**Comment by bgruening** [comment]
Are you planning to fix the linting?

**Comment by RZ9082** [comment]
> Are you planning to fix the linting?

sure.

File: tools/mcl/mcl.xml
Comment (RZ9082) [inline tools/mcl/mcl.xml:1]: removed References from help, the citation and bio.tools ref should be enough right?

File: tools/mcl/mcl.xml [hunk 1]
```diff
@@ -77,35 +80,14 @@ vary this parameter to obtain clusterings at different levels of granularity. A
...
-
-and add proper attributions to the MCL-edge home page http://micans.org/mcl/. For biological applications, it is appropriate to cite, additionally, the reference article for the first application of MCL to biological data:
-
-Enright A.J., Van Dongen S., Ouzounis C.A., An efficient algorithm for large-scale detection of protein families, Nucleic Acids Research 30(7):1575-1584 (2002).
-
-
 **Galaxy Wrapper Author**::
 
     *  Bjoern Gruening, University of Freiburg
     *  Pavan Videm , University of Freiburg
```
Comment (bgruening) [inline tools/mcl/mcl.xml:86]: That could also be converted into https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-person

**Comment by bgruening** [comment]
Sorry, those tools have been migrated to IUC long time ago :(

I removed them now from this repo, should have done that a wile back. Please check IUC.

**Pull request merged**